tools/ci/check_grep.sh

```diff
@@ -51,6 +51,10 @@ if grep -i 'centcomm' _maps/**/*.dmm; then
     echo "ERROR: Misspelling(s) of CENTCOM detected in maps, please remove the extra M(s)."
     st=1
 fi;
+if grep 'set name\s*=\s*"[\S\s]*![\S\s]*"' code/**/*.dm; then
```

Comment:
Would it not be better to use .?

Might be worth editing one of the code lines quickly to fail it and then revert it

Reply:
`.` matches everything except a line break, `\s` includes line breaks. This version is more resilient in the edge case someone decides to add line breaks - it doesn't really matter either way.

I guess I can trigger it.